File: sample-conference-videochat/sample-conference-videochat/Components/Settings/SliderItemModel.m
//
//  SliderItemModel.m
//  sample-videochat-webrtc
//
//  Created by Andrey Ivanov on 30/09/15.
//  Copyright © 2015 QuickBlox Team. All rights reserved.
//

#import "SliderItemModel.h"
#import "SettingSliderCell.h"

@implementation SliderItemModel

- (Class)viewClass {
    
    return [SettingSliderCell class];
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/Settings/SliderItemModel.swift
//
//  SliderItemModel.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 12/11/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import Foundation

class SliderItemModel: BaseItemModel {
    //MARK: - Properties
    var minLabel = ""
    var maxLabel = ""
    var maxValue: Int = 0
    var currentValue: UInt = 0
    var minValue: Int = 0
    var isDisabled = false
    
    //MARK: - Overrides
    override func viewClass() -> AnyClass {
        return SettingSliderCell.self
    }
}
